File: lib-model/src/main/java/com/minesec/msa/client/sdk/management/model/DeviceInfoData.java
package com.minesec.msa.client.sdk.management.model;

import java.util.List;

import lombok.Getter;

/**
 * @author eric.song
 * @since 2023/4/20 11:18
 */
@Getter
public class DeviceInfoData {
    public String sysVersion;
    public String transType;
    public String groupId;
    public String mchName;
    public String mcc;
    public String uuid;
    public String helpSupportUrl;
    public String currency;
    public String mchLogo;
    public String mchContactAddress;
    public boolean loadEmvConfig;
	public boolean enableExternalReader;
	public List<String> supportedMPQRMethods; //supported QR payment methods.
}

File: lib-model/src/main/java/com/minesec/msa/client/sdk/management/model_k/DeviceInfoData_.kt
package com.minesec.msa.client.sdk.management.model_k

/**
 * @author eric.song
 * @since 2024/6/18 14:20
 */
data class DeviceInfoData_(
    val sysVersion: String,
    val transType: String,
    val groupId: String,
    val mchName: String,
    val mcc: String,
    val uuid: String,
    val helpSupportUrl: String,
    val currency: String,
    val mchLogo: String,
    val mchContactAddress: String,
    val loadEmvConfig: Boolean,
    val enableExternalReader: Boolean,
    val supportedMPQRMethods: List<String>
)
